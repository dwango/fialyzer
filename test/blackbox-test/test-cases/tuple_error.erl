-module(tuple_error).

-export([main/0]).

main() ->
    {ok, X} = {1, 2},
    X.
