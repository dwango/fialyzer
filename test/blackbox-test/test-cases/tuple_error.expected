test-cases/tuple_error.erl:6: Type error: type mismatch;
  found   : {1, 2}
  required: {'ok', any()}
there is no solution that satisfies subtype constraints
